travis/build_info.py

```diff
@@ -0,0 +1,207 @@
+"""
+Gather Travis usage info in real-time using Travis REST APIs
+
+This only applies to travis instances that do not require
+authorization (e.g., travis-ci.org). Auth is a #TODO
+
+"""
+
+import argparse
+import logging
+import requests
+import sys
+
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = 'https://api.travis-ci.org/'
+
+
+def get_repos(org):
+    """
+    Returns list of active repos in a given org.
+    """
+
+    repo_list = []
+    # Note: One call is needed if org has <= 100 active repos.
+    # If org has > 100, this code will need to be modified
+    #  to include 'offset' requests
+    req = requests.get(
+        BASE_URL + 'v3/owner/{org}/repos?active=true'.format(org=org)
+    )
+    if req.status_code != 200:
+        raise requests.HTTPError(req.status_code)
+    req_json = req.json()
+    try:
+        for i in req_json['repositories']:
+            repo_list.append(i['name'])
+    except KeyError:
+        raise KeyError("Cannot parse response")
+
+    return repo_list
+
+
+def get_active_builds(org, repo):
+    """
+    Returns list of active builds for a given repo slug
+    """
+
+    repo_slug = '{org}/{repo}'.format(org=org, repo=repo)
+    req = requests.get(
+        BASE_URL + 'repos/{repo_slug}/builds'.format(repo_slug=repo_slug)
+    )
+    build_list = req.json()
+    active_build_list = []
+    for build in build_list:
+        if build['state'] != 'finished':
```

Comment:
I recommend that we either put statements that refer directly to keys in try blocks and catch KeyErrors or use the get() method for dicts, in this method and others. E.g. build.get('state')